Packs/RsaNetWitnessPacketsAndLogs/Integrations/RSANetWitnessPacketsAndLogs/RSANetWitnessPacketsAndLogs.js

```diff
@@ -10,8 +10,10 @@ var NON_SSL_PORT = '50105';
 var LAST_HOURS = 'lastHours';
 var LAST_MINUTES = 'lastMinutes';
 var QUERY = 'query';
-var USER_NAME = params.username;
-var PASSWORD = params.password;
+
+var USER_NAME = params.username || params.credentials.identifier;
+var PASSWORD = params.password || params.credentials.password;
```

Comment:
```suggestion
var USER_NAME =  params.credentials.identifier || params.username;
var PASSWORD = params.credentials.password || params.password;
```

my bad, when we have the new version, there is no need to check for the old.